experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -92,6 +97,47 @@ export function generateTestTrees(): { name: string; tree: ISharedTree }[] {
 					);
 				};
 
+				const provider = new TestTreeProviderLite(1, new SharedTreeTestFactory(onCreate));
+				return provider.trees[0];
+			},
+		},
+		{
+			name: "nested-sequence-change",
+			tree: () => {
+				const onCreate = (tree: ISharedTree) => {
+					const rootFieldSchema = fieldSchema(FieldKinds.sequence);
+					const rootNodeSchema = namedTreeSchema({
+						name: brand("Node"),
+						mapFields: fieldSchema(FieldKinds.sequence),
+					});
+					const testSchema: SchemaData = {
+						treeSchema: new Map([[rootNodeSchema.name, rootNodeSchema]]),
+						rootFieldSchema,
+					};
+					tree.storedSchema.update(testSchema);
```

Comment:
nit: Can you use a schema builder and tree.schematize to create the schema rather than doing it manually? You don't need a schema aware tree for this test but it still seems nice to use the higher-level APIs to accomplish the same thing (if they can).

Reply:
Thanks, I had not looked those higher level APIs before.